lib/utils/explain-dep.js

```diff
@@ -88,18 +107,23 @@ const explainDependents = ({ name, dependents }, depth, color) => {
 
 const explainEdge = ({ name, type, bundled, from, spec }, depth, color) => {
   const { bold } = color ? chalk : nocolor
+  const dep = type === 'workspace'
+    ? bold(relative('.', spec.slice('file:'.length)))
```

Comment:
```suggestion
    ? bold(relative(edge.from.root.realpath, spec.slice('file:'.length)))
```

This really needs to be a specific path, I think, not `.`, or else the tests are dependent on the working directory that they're run in.

Reply:
Oh, right, but edge is not an actual Arborist.Edge, it's a `node.explain().dependents` entry.  🤔

Reply:
This works:

```js
bold(relative(from.location, spec.slice('file:'.length)))
```

Reply:
💯